pkg/emb6/patches/0004-Provide-event-type-for-RIOT-s-sock_.-_send.patch

```diff
@@ -26,13 +26,13 @@ index 5316a67..78153be 100644
 +#define EVENT_TYPE_SOCK_SEND        0x08    ///< Send command from sock layer
  #define OBLIG_EVENT_PRIOR           0x0a
  #define EVENT_TYPE_PCK_LL           0x0a    ///< New low level packet received
-
-
+ 
+ 
 -#define EVENT_TYPES_COUNT           8       ///< Counter of events in /ref EVENT_TYPES macro
 +#define EVENT_TYPES_COUNT           9       ///< Counter of events in /ref EVENT_TYPES macro
  #define MAX_CALLBACK_COUNT          7       ///< Maximal amount of callbacks in /ref st_funcRegList_t list
  #define EVPROC_QUEUE_SIZE           20      ///< Maximal amount of events in /ref pst_evList queue
  /*=============================================================================
---
-1.9.1
+-- 
+2.16.1
```

Comment:
Apart from the required whitespaces and hash-updates from length 7 to 8 this version string seems to be the only required change. Can you try if only the whitespace changes fix the problem for you?

Reply:
as said, I simply regenerated the patches completely and even so some changes (as this one) are unnecessary, and yes the hash-updates are most likely what fixes the problem.